include/api/ofp_init.h

```diff
@@ -430,6 +430,17 @@ void ofp_stop_processing(void);
 
 odp_bool_t *ofp_get_processing_state(void);
 
+/**
+ * Get OFP global parameters
+ *
+ * @param params Structure to be filled with OFP global parameters
+ *
+ * @retval 0 on success
+ * @retval -1 on failure
+ */
+
+int ofp_get_global_parameters(ofp_global_param_t *params);
```

Comment:
Add comment when this can be called (now after ofp_global_init() but I think after ofp_local_init() would be more right (and would make the implementation slightly simpler).

Maybe the name could be ofp_get_global_param() for consistensy with ofp_init_global_param().